repository-data/webfiles/src/main/resources/site/src/scss/components/_utils.scss

```diff
@@ -12,6 +12,10 @@
     padding-top: 0;
 }
 
+.no-padding {
+    padding: 0;
+}
+
```

Comment:
Would `nhsd-!t-padding-0` from the UIToolkit work?

Reply:
I am afraid our knowledge of the UI is very limited and tackling this may delay the release. Is it ok if we do not fix this now?